framework/source/class/qx/bom/client/Event.js

```diff
@@ -32,21 +32,19 @@ qx.Bootstrap.define("qx.bom.client.Event",
     /**
      * Checks if touch events are supported.
      *
-     * @internal
```

Comment:
@schnepe2 I think this is still true: this methods should only be used internally. Or does this generate a warning in the API viewer?